docs/contributing/stacks.md

```diff
@@ -1,13 +1,13 @@
 # Community Stacks
```

Comment:
IMHO this document is long enough it may be useful to break into separate pages the way zero to jupyterhub does https://zero-to-jupyterhub.readthedocs.io/en/latest/google/step-zero-gcp.html

Reply:
I agree it's a bit long, and will probably be longer in a few moments when I add screenshots. 

If we can nest the pages for this particular document in the sidebar ToC somehow, then I'm all for splitting it up. (Do you know a way to do it?) If not, I think a single page with the navigation by heading might be a better choice to ensure users know to work through all of the steps.

Reply:
z2jh does it @willingc or @choldgraf may have suggestions

Reply:
IMO since the RTD theme you're using already uses navigation by header, I think it's OK. Though if it grows a lot longer and/or has deeper sub-headings, then we could revisit splitting into multiple pages.

![image](https://user-images.githubusercontent.com/1839645/40963235-cabce2c6-689f-11e8-802d-62c4455c6f66.png)